cloud-gateway-service/src/test/java/org/zowe/apiml/cloudgatewayservice/service/scheme/HttpBasicPassticketTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.cloudgatewayservice.service.scheme;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.cloud.gateway.filter.FilterDefinition;
+import org.springframework.cloud.gateway.route.RouteDefinition;
+import org.zowe.apiml.auth.Authentication;
+import org.zowe.apiml.auth.AuthenticationScheme;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class HttpBasicPassticketTest {
+
+    @Test
+    void givenInstance_whenGetAuthenticationScheme_thenReturnProperType() {
+        assertEquals(AuthenticationScheme.HTTP_BASIC_PASSTICKET, new HttpBasicPassticket().getAuthenticationScheme());
+    }
+
+    @Test
+    void givenInstance_whenApply_thenFulfillFilterFactorArgs() {
```

Comment:
what is the reason of having Instance as the assumption?

Reply:
givenRouteDefinition_whenApply_thenFulfillFilterFactorArgs